### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by ahohnsen**

### CODE REVIEW

File: src/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+
+export default function Navigation() {
+  return (
+    <StyledFooter>
```
Comment (mwarnck) [inline src/Navigation.js:6]: do you really need the footer around the nav? Isn´t it possible to the style the nav directly like you have done it below?

**Review by mwarnck — APPROVED** [review]
I approve your work, well done. Only two comments you can have a look at ;)

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,42 @@
...
 export default function App() {
   const [diveWishes, setDiveWishes] = useState([]);
 
   return (
-    <>
-      <Heading>Diving Wishlist</Heading>
-      <AddDiveWish onAddDiveWish={addToWishlist} />
-      <Wishlist diveWishes={diveWishes} />
-    </>
+    <AppGrid>
```
[the remark this answers is not in the document]
Comment (ahohnsen) [reply src/App.js:12]: Research online told me it is possible :) I cannot think of another way to do that right now....

File: src/Navigation.js (hunk 1 above)
Comment (ahohnsen) [reply src/Navigation.js:6]: I added the footer around the nav as I want to have a bar on the bottom of the viewport when the app is viewed on the desktop. But the nav should not be spread over the full width. So in this way it looks fine either way - desktop or mobile. Maybe you have another idea to solve that?

**Pull request merged**